Code/Core/Mem/MemDebug.cpp

```diff
@@ -40,12 +40,20 @@ void MemDebug::FillMem( void * ptr, const size_t size, const uint32_t pattern )
                             (char)( ( pattern & 0x00FF0000 ) >> 16 ) };
         const char * b = bytes;
         char * cit = static_cast< char * >( static_cast< void * >( it ) );
+
+        #if defined( __GNUC__ ) && __GNUC__ >= 7
+            #pragma GCC diagnostic push
+            #pragma GCC diagnostic ignored "-Wimplicit-fallthrough"
+        #endif
```

Comment:
This switch statement has now been changed so this warning disabling is no longer required: https://github.com/fastbuild/fastbuild/commit/68e5e7dc88774d63dd6c81f67b2ae64b7570f565